cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/internal/documentstore/DocumentRevisionBuilder.java

```diff
@@ -28,26 +28,6 @@
  */
 public class DocumentRevisionBuilder {
 
-
```

Comment:
It was completely unused. I think it stems from a time before we allowed users to `new` up `DocumentRevisions`